rally_ovs/plugins/ovs/scenarios/ovn.py

```diff
@@ -639,3 +639,42 @@ def _get_address_set(self, set_name):
 
         ovn_nbctl = self._get_ovn_controller(self.install_method)
         return ovn_nbctl.get("Address_Set", set_name, 'addresses')
+
+    def runCmd(self, ssh, cmd=""):
```

Comment:
I think we should add the `background_opt` argument here instead of doing the same work twice in `runFarmCmd` and `runControllerCmd`. Same for `pid_opt` (and potentially `pid_proc_name` if we add it - see below).

Reply:
@dceara ack, I will update it